tsl/test/expected/dist_move_chunk.out

```diff
@@ -335,6 +321,208 @@ FROM timescaledb_experimental.chunk_replication_status;
+           time           | device | temp 
+--------------------------+--------+------
+ Thu Mar 08 00:00:00 2018 |     10 |  0.1
+ Thu Mar 08 01:00:00 2018 |      1 |  0.1
+(2 rows)
+
+-- Get compressed chunk name on the source data node and show its content
+\c :DN_DBNAME_1 :ROLE_CLUSTER_SUPERUSER;
+SELECT c2.table_name
+FROM _timescaledb_catalog.chunk c1
+JOIN _timescaledb_catalog.chunk c2 ON (c1.compressed_chunk_id = c2.id)
+WHERE c1.table_name = '_dist_hyper_3_12_chunk';
+        table_name        
+--------------------------
+ compress_hyper_3_6_chunk
+(1 row)
+
+SELECT * FROM _timescaledb_internal.compress_hyper_3_6_chunk;
+                           time                           | device |                                                                       temp                                                                       | _ts_meta_count | _ts_meta_sequence_num |      _ts_meta_min_1      |      _ts_meta_max_1      
+----------------------------------------------------------+--------+--------------------------------------------------------------------------------------------------------------------------------------------------+----------------+-----------------------+--------------------------+--------------------------
+ BAAAAgna9rUEAAACCdr2tQQAAAAAAQAAAAEAAAAAAAAADgAEE7XtaggA |      1 | AwA/uZmZmZmZmgAAAAEAAAABAAAAAAAAAAEAAAAAAAAAAQAAAAEAAAABAAAAAAAAAAEAAAAAAAAAAQAAAAEGAAAAAAAAAAIAAAABAAAAAQAAAAAAAAAGAAAAAAAAAD0AAAABPR/czMzMzMzN |              1 |                    10 | Thu Mar 08 01:00:00 2018 | Thu Mar 08 01:00:00 2018
+ BAAAAgnaICFgAAACCdogIWAAAAAAAQAAAAEAAAAAAAAADgAEE7RAQsAA |     10 | AwA/uZmZmZmZmgAAAAEAAAABAAAAAAAAAAEAAAAAAAAAAQAAAAEAAAABAAAAAAAAAAEAAAAAAAAAAQAAAAEGAAAAAAAAAAIAAAABAAAAAQAAAAAAAAAGAAAAAAAAAD0AAAABPR/czMzMzMzN |              1 |                    20 | Thu Mar 08 00:00:00 2018 | Thu Mar 08 00:00:00 2018
+(2 rows)
+
+SELECT * FROM _timescaledb_internal._dist_hyper_3_12_chunk ORDER BY time;
+           time           | device | temp 
+--------------------------+--------+------
+ Thu Mar 08 00:00:00 2018 |     10 |  0.1
+ Thu Mar 08 01:00:00 2018 |      1 |  0.1
+(2 rows)
+
+-- Move compressed chunk from data node 1 to data node 2
+\c :TEST_DBNAME :ROLE_CLUSTER_SUPERUSER;
+CALL timescaledb_experimental.move_chunk(chunk=>'_timescaledb_internal._dist_hyper_3_12_chunk', source_node=> 'data_node_1', destination_node => 'data_node_2');
+SELECT count(*) FROM  _timescaledb_catalog.chunk_copy_operation;
+ count 
+-------
+     5
+(1 row)
+
+-- Make sure same compressed chunk hash been created on the destination data node
+\c :DN_DBNAME_2 :ROLE_CLUSTER_SUPERUSER;
+-- Chunk created on data node has different id but the same name, make sure
+-- compressed_chunk_id is correctly set
+SELECT c2.table_name
+FROM _timescaledb_catalog.chunk c1
+JOIN _timescaledb_catalog.chunk c2 ON (c1.compressed_chunk_id = c2.id)
+WHERE c1.table_name = 'compress_hyper_3_12_chunk';
+ table_name 
```

Comment:
This should return a compressed chunk name, right?

Reply:
Good catch, this should be `_dist_hyper_3_12_chunk` but query should return `compress_hyper_3_6_chunk`